### PR TITLE
SSH secret deployment to drupal-ckeditor-libraries-group

### PR DESCRIPTION
**Pull request by sembrat**
I believe I may have this working, although I want to get a second set of eyes on it.

SSH key is defined in `action_runner` -> Secrets, which is onboarded and use to commit to the repo using user `drupal-ckeditor-libraries-group-machine`.

Code changes test on https://github.com/drupal-ckeditor-libraries-group/templates .

### CODE REVIEW

**Comment by sembrat** [comment]
Requesting squash and rebase. 

This will also need to replace the other repos back into the matrix in `manual.yml` once this is in place.

**Comment by sembrat** [comment]
I do want to sanity check though... is this properly pulling releases and commits? Requesting a check.

**Comment by sembrat** [comment]
Well I'm an idiot - `actions/checkout` by default pulls only one commit. Adjusted the parameters and running tests on the `templates` project to see if it works.

**Comment by sembrat** [comment]
Confirming it now pulls commits for the folder. Now to determine why it isn't rebasing release tags based on this...

**Comment by sembrat** [comment]
Tags are in, so I believe this is good to go pending review.

File: template/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (iVegas) [inline template/package.json]: Looks like we move the template from the root of the repo. Can we remove the template from the repo root?

File: .github/workflows/manual.yml [hunk 2]
```diff
@@ -12,16 +12,12 @@ on:
...
       matrix:
         repo_name:
-          # define the list of repos here
-          - codesnippet
-          - colorbutton
-          - fakeobjects
-          - font
-          - notification
+        # define the list of repos here
+        - templates
```
Comment (iVegas) [inline .github/workflows/manual.yml:25]: Looks like we can revert changes here and just add `templates` to the list. And keep alphabetical order)

**Review by iVegas — CHANGES_REQUESTED** [review]

File: .github/workflows/manual.yml (hunk 2 above)
Comment (sembrat) [reply .github/workflows/manual.yml:25]: Resolved: https://github.com/drupal-ckeditor-libraries-group/actions_runner/pull/4/commits/f81944eb6cc769c253af60876128c9531403dc99

File: template/package.json (hunk 1 above)
Comment (sembrat) [reply template/package.json]: Where would you like these stored? Happy to toss them wherever. @iVegas

Comment (iVegas) [reply template/package.json]: Sorry, nevermind. I just realized that we moved into the template folder recently.

**Review by iVegas — APPROVED** [review]

**Pull request merged**